src/dealWizard/makeOfferWizard/makeOfferWizard.ts

```diff
@@ -0,0 +1,47 @@
+import { autoinject } from "aurelia-framework";
```

Comment:
This comment is not specific to this place.
It's just to have thread commenting

---

Higher level notes:
_(just some observations, which may be no issue. These notes are rather for me to get an understanding what is happening here)_

- `makeOfferWizard.ts` and `openDealWizard.ts` same except for one stage
  - their views are also the same

- Each wizard skeleton:
  - pstepper
  - router-view
    - stage-wrapper
      - stage-header
      - stage-content
        - `<slot>`

- Example: Make offer wizard
  - pstepper
  - router-view (4 stages)
    - stage-wrapper
      - stage-header
      - stage-content
        - `<slot>`
          - ...
          - `<partner-dao-stage>`   (<-- Example)
            - `<dao-details-section>`  (<-- resuable for `primary-dao-stage`)


## Unrelated
- pform-input -> pform-field
(because <pform-input><ptextarea><...> reads strangely)